rs/tests/driver/src/driver/prometheus_vm.rs

```diff
@@ -271,6 +272,10 @@ fi
             )
             .unwrap();
 
+        let grafana_dashboards_dst = config_dir.join("grafana").join("dashboards");
+        if !grafana_dashboards_dst.exists() {
```

Comment:
According to [the docs](https://doc.rust-lang.org/beta/std/fs/fn.create_dir_all.html) `std::fs::create_dir_all` doesn't error when the specified path already exists so I think you don't need this if-statement.